Semantic Scholar.js

```diff
@@ -37,148 +37,423 @@
+				"creators": [
+					{
+						"firstName": "Dzmitry",
+						"lastName": "Bahdanau",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Kyunghyun",
+						"lastName": "Cho",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Yoshua",
+						"lastName": "Bengio",
+						"creatorType": "author"
+					}
+				],
+				"date": "1 September 2014",
+				"abstractNote": "Neural machine translation is a recently proposed approach to machine translation. Unlike the traditional statistical machine translation, the neural machine translation aims at building a single neural network that can be jointly tuned to maximize the translation performance. The models proposed recently for neural machine translation often belong to a family of encoder-decoders and consists of an encoder that encodes a source sentence into a fixed-length vector from which a decoder generates a translation. In this paper, we conjecture that the use of a fixed-length vector is a bottleneck in improving the performance of this basic encoder-decoder architecture, and propose to extend this by allowing a model to automatically (soft-)search for parts of a source sentence that are relevant to predicting a target word, without having to form these parts as a hard segment explicitly. With this new approach, we achieve a translation performance comparable to the existing state-of-the-art phrase-based system on the task of English-to-French translation. Furthermore, qualitative analysis reveals that the (soft-)alignments found by the model agree well with our intuition.",
+				"libraryCatalog": "Semantic Scholar",
+				"publicationTitle": "CoRR",
+				"url": "https://www.semanticscholar.org/reader/fa72afa9b2cbc8f0d7b05d52548906610ffbb9c5",
+				"attachments": [
+					{
+						"title": "Semantic Reader Link",
+						"mimeType": "text/html",
+						"snapshot": false
+					},
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [
+					"[TLDR] It is conjecture that the use of a fixed-length vector is a bottleneck in improving the performance of this basic encoder-decoder architecture, and it is proposed to extend this by allowing a model to automatically (soft-)search for parts of a source sentence that are relevant to predicting a target word, without having to form these parts as a hard segment explicitly."
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.semanticscholar.org/reader/2fd10e095b146f99da8cdc6ff58720e2e8fca36d",
+		"items": [
+			{
+				"itemType": "conferencePaper",
+				"title": "When Attention Meets Fast Recurrence: Training Language Models with Reduced Compute",
+				"creators": [
+					{
+						"firstName": "Tao",
+						"lastName": "Lei",
+						"creatorType": "author"
+					}
+				],
+				"date": "24 February 2021",
+				"DOI": "10.18653/v1%2F2021.emnlp-main.602",
```

Comment:
I don't think so. This seems directly coming from the schema-metadata of Semantic Scholar - I'll cut a ticket there to fix it.

![image](https://user-images.githubusercontent.com/84034381/199577675-de644c6d-2276-4172-91c8-d2d08badbb56.png)


For now, does it sound good to you if I use another test case and assume the DOI format will be fixed by the end of this week?

Reply:
Sorry I was wrong. Calling `decodeURIComponent` is the right way to go since the `sameAs` field holds a DOI link instead of a DOI tag. The value under `<meta name="dc.identifier">` is not necessarily a DOI. When a paper doesn't have a DOI, Semantic Scholar will put an identifier under the field. Here is an example:
<img width="1305" alt="image" src="https://user-images.githubusercontent.com/84034381/199600965-4eb84b5f-058e-4ed0-ac35-794986f1f599.png">